en/components/texthighlight.md

```diff
@@ -0,0 +1,344 @@
+    public index: number = 0;
+
+
+    public get canMoveHighlight() {
+        return this.matchCount > 1;
+    }
+```
+
+Then we need to add the following methods which will allow the user to apply the highlights for the text they have typed in the search box and to move the active highlight around.
+
+``` typescript
+     public searchKeyDown(ev) {
+        if (this.searchText) {
+            if (ev.key === "Enter" || ev.key === "ArrowDown" || ev.key === "ArrowRight") {
+                ev.preventDefault();
+                this.findNext();
+            } else if (ev.key === "ArrowUp" || ev.key === "ArrowLeft") {
+                ev.preventDefault();
+                this.findPrev();
+            }
+        }
+    }
+
+    public onTextboxChange() {
+        this.index = 0;
+        this.find(0);
+    }
+
+    public updateSearch() {
+        this.caseSensitive = !this.caseSensitive;
+        this.find(0);
+    }
+
+    public clearSearch() {
+        this.searchText = "";
+        this.find(0);
+    }
+
+    private findNext() {
+        this.find(1);
+    }
+
+    private findPrev() {
+        this.find(-1);
+    }
+
+    private find(increment: number) {
+        if (this.searchText) {
+            this.matchCount = this.highlight.highlight(this.searchText, this.caseSensitive);
+            this.index += increment;
+
+            this.index = this.index < 0 ? this.matchCount - 1 : this.index;
+            this.index = this.index > this.matchCount - 1 ? 0 : this.index;
+
+            if (this.matchCount) {
+                IgxTextHighlightDirective.setActiveHighlight("group1", {
+                    columnIndex: 0,
+                    index: this.index,
+                    page: 0,
+                    rowIndex: 0
+                });
+            }
+        } else {
+            this.highlight.clearHighlight();
+        }
+    }
+```
+
+If the sample is configured properly, the final result should look like that:
+
+<div class="sample-container loading" style="height: 260px;">
+    <iframe id="text-highlight-1-iframe" src='{environment:demosBaseUrl}/text-highlight-1' width="100%" height="100%" seamless
+        frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+<button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="text-highlight-1-iframe"
+    data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+<div class="divider"></div>
+
+### Search across multiple containers
+The igxTextHighlight allows you to search across multiple containers and which would all share one active highlight. This is done by having the same group value across multiple TextHighlight directives which have all separate containers. In order to setup the sample we will reuse the search box from the previous sample, but this time we will add two paragraphs. Again, note that they both are in their own containers, but this time the second one has a different row value.
```

Comment:
"... multiple containers and which would all share ..." probably the "and" should be removed.